cpp/src/plasma/malloc.h

```diff
@@ -35,6 +37,14 @@ void GetMallocMapinfo(void* addr, int* fd, int64_t* map_length, ptrdiff_t* offse
 /// @return The size of the corresponding memory-mapped file.
 int64_t GetMmapSize(int fd);
 
-void SetMallocGranularity(int value);
+struct mmap_record {
```

Comment:
Can't you put all this in a namespace?

Reply:
Yes, I put everything in a namespace now!